packages/labs/cli-mcp-server/src/index.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import type {ResolvedCommand} from '@lit-labs/cli';
+import {run} from './run.js';
+import type {McpOptions} from './types.js';
+
+/**
+ * Defines our interface within the Lit CLI.
+ */
+export const getCommand = (): ResolvedCommand => {
+  return {
+    kind: 'resolved',
+    name: 'mcp',
+    description: 'Lit MCP server',
+    options: [
+      {
+        name: 'environment',
+        type: String,
+        alias: 'E',
+        defaultValue: 'local',
+        description:
+          'The environment to run in, either "local" or "server". Server mode will ' +
+          'only expose MCP commands that are available on the server which are commands that do ' +
+          'not require local file access.',
+        typeLabel: '"local", "server"',
```

Comment:
What local filesystem tools are you thinking of?

Reply:
This is what we were discussing previously:

- Local, requires filesysten access like generating / finding CE manifests
- Server, only requires network access to enable a RAG via MCP over HTTP (such as site search)